editor/src/clj/editor/app_view.clj

```diff
@@ -1484,6 +1484,8 @@ If you do not specifically require different script states, consider changing th
 (handler/defhandler :reload-stylesheet :global
   (run [] (ui/reload-root-styles!)))
 
+; This function re-implemented in Java in Start.java->newWindow(). If you change something
+; please, change it there as well.
```

Comment:
I would prefer to reuse java code in Clojure. But I think it's not something we want most of the time. So I just added this comment here to make sure we keep it sync.